### DISCUSSION TITLE
Is it possible to add a one-click clear history button?

### DISCUSSION THREAD

**Discussion by kidzgy** [sumatrapdfreader/sumatrapdf#4106] [Ideas]
Currently to clear the history, you can only delete them one by one by right clicking on the home page, I wonder if it is possible to add a button that can clear all the history with one click?
Thanks!

**Comment by GitHubRulesOK** [comment]
no need it is easily done via menu
uncheck remember files and click OK repeat with enabled if required
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/91c81af5-1008-4f76-84da-3f4c76494f4e)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/50e12157-0b0a-4eaf-8899-545b2fec4818)

It will only remember favorites from before
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7f0eccae-bec8-4337-ab88-53b4156363fa)

**Comment by kidzgy** [comment]
This is different, adding a one-click delete history button does not conflict with whether or not to remember history.

The idea is that when I need to remember history normally, I can easily delete it with one click when I don't need it.

If in your case, if I want to delete history with one click, then I need to turn off remember history and then turn it back on again.

**Comment by kjk** [comment]
You can delete history quickly with [command palette](https://www.sumatrapdfreader.org/docs/Command-Palette): `Ctrl + K` and `clear history` (need to type only part of that to narrow down and `Enter` to execute the command)

**Reply by GitHubRulesOK** [reply]
@kidzgy 
My bad ! As I should have remembered from a similar question
 so a single key approach should be possible using a key assigned by advanced options  "Shortcut"s
